protocol/messenger.go

```diff
@@ -2241,11 +2244,6 @@ func (m *Messenger) sendChatMessage(ctx context.Context, message *common.Message
 		message.Payload = &protobuf.ChatMessage_Community{Community: wrappedCommunity}
 
 		message.ContentType = protobuf.ChatMessage_COMMUNITY
-	} else if len(message.AudioPath) != 0 {
-		err := message.LoadAudio()
-		if err != nil {
-			return nil, err
-		}
```

Comment:
The changes above can be reverted

Reply:
Done. Check consolidated changes in dace1b28d8510bc9514ff4661928d2fb5c0302e6.